code/modules/events/spacevine.dm

```diff
@@ -197,7 +205,7 @@
 
 /datum/spacevine_mutation/aggressive_spread/on_spread(obj/effect/spacevine/holder, turf/target)
 	target.ex_act(severity)
-	for(var/atom/A in target)
+	for(var/atom/A in target.contents + target)

```

Comment:
I forgot about this line
Actually this will be 3 times since turfs ex_act calls it on all things inside anyway